server/src/main/scala/ch.epfl.scala.index.server/routes/api/SearchApi.scala

```diff
@@ -91,7 +104,8 @@ class SearchApi(
   val routes: Route =
     pathPrefix("api") {
       cors() {
-        path("search") {
+        // deprecated endpoint replaced by api/search endpoint
+        path("search-old") {
```

Comment:
Should we instead keep this one and introduce a new endpoint `api/search/v2`, to not break existing users?